frontend/src/pages/_app.tsx

```diff
@@ -131,6 +131,7 @@ function App({ Component, pageProps }: AppPropsWithLayout): JSX.Element {
         data-domain={configs.PLAUSIBLE_DATA_DOMAIN}
```

Comment:
ideally we would want to remove this and replace it with the `src="/js/script.js"` one, but i think it would be better to verify the proxy works in all environments before removing this. i don't believe there should be any harm to embedding the script twice